src/DataConsumer.ts

```diff
@@ -316,9 +349,29 @@ export class DataConsumer extends EnhancedEventEmitter
 					break;
 				}
 
+				case 'bufferedamount':
+				{
+					const { bufferedAmount } = data as { bufferedAmount: number };
+
+					if (
+						bufferedAmount <= this._bufferedAmountLowThreshold &&
+						bufferedAmount <= this._bufferedAmount
+					)
+					{
+						this.safeEmit('bufferedamountlow', bufferedAmount);
+
+						// Emit observer event.
```

Comment:
Let's not emit this event in the observer, please. Anyway it's not documented within the `observer` inline doc, but better do not emit this in observer.

Reply:
done